### PR TITLE
Changes needed to get build working for nucleo board

### PR DESCRIPTION
**Pull request by chris-hl**

### CODE REVIEW

File: zephyr/CMakeLists.txt [hunk 1]
```diff
@@ -120,4 +121,5 @@ zephyr_library_sources(
   ${ZEPHYR_CURRENT_MODULE_DIR}/rmw_microxrcedds_c/src/rmw_microros/time_sync.c
   ${ZEPHYR_CURRENT_MODULE_DIR}/rmw_microxrcedds_c/src/rmw_microros/ping.c
   ${ZEPHYR_CURRENT_MODULE_DIR}/rmw_microxrcedds_c/src/rmw_microros/timing.c
+  $<$<BOOL:${RMW_UXRCE_TRANSPORT_CUSTOM}>:${ZEPHYR_CURRENT_MODULE_DIR}/rmw_microxrcedds_c/src/rmw_microros/custom_transport.c>
```
Comment (yashi) [inline zephyr/CMakeLists.txt]: Please make it Kconfig.

**Review by yashi — CHANGES_REQUESTED** [review]

File: zephyr/CMakeLists.txt (hunk 1 above)
Comment (cj-hl) [reply zephyr/CMakeLists.txt]: Given that the transport is hardwired at the moment. I'm tempted to simply remove the conditional from this line, and also remove the unused lines 62,63 (setting UDP addr and port).

Once we have a good pattern for converting between zephyr Kconfig and ros cmake config, we can do it properly.

Comment (yashi) [reply zephyr/CMakeLists.txt]: love that idea.

Comment (cj-hl) [reply zephyr/CMakeLists.txt]: done

**Pull request merged**